src/main/java/com/jcabi/xml/SaxonDocument.java

```diff
@@ -86,6 +93,56 @@ public SaxonDocument(final String text) {
         this(SaxonDocument.node(text));
     }
 
+    /**
+     * Public constructor from XML as byte array.
+     * @param data XML document body as byte array.
```

Comment:
@volodya-lombrozo it's a good practice to add `@since` to a method or a class when you introduce it to the code base: https://stackoverflow.com/questions/578363/javadoc-version-and-since

Reply:
@yegor256 I've added the "since" tag